README.md

```diff
@@ -478,6 +483,76 @@ pk.patch.deleteLang('<patch-id>', 'fr');
 NOTES:
 - Deleting a translation will return a `409` error if there is no default language and no other translation available.
 
+### `pk.keys.list()`
+
+⚠️ Restricted to **private** API keys, **do NOT expose the private key to the browser**.
+
+Retrieve all application API keys.
+
+```js
+pk.keys.list();
+```
+
+### `pk.keys.create()`
+
+⚠️ Restricted to **private** API keys, **do NOT expose the private key to the browser**.
+
+Create an application API key.
+
+```js
+pk.keys.create('<role>', { domains: [] });
+```
+
+| Parameter | Type | Description |
+| --- | --- | --- |
+| `role` | `('public' \| 'private')` | API key role. |
+| `options` | `object?` | API key options. |
+| `options.domains` | `string[]?` | Domain or IP restriction (for public keys only). |
+
+### `pk.keys.get()`
+
+⚠️ Restricted to **private** API keys, **do NOT expose the private key to the browser**.
+
+Retrieve an API key by ID.
+
+```js
+pk.keys.get('<key-id>');
+```
+
+| Parameter | Type | Description |
+| --- | --- | --- |
+| `id` | `string` | API key ID. |
+
+### `pk.keys.update()`
+
+⚠️ Restricted to **private** API keys, **do NOT expose the private key to the browser**.
+
+Update an API key.
```

Comment:
`Update the domains list of the API key`

Reply:
Yeah for now, but left the `create` and `update` methods open to more options. I wouldn't want to have breaking changes the fay we add an option other than `domains`, we never know.